pages/tech-partners/style.module.css

```diff
@@ -0,0 +1,128 @@
+.posts {
+  display: grid;
+  grid-gap: 40px;
```

Comment:
Replace with `gap`. `grid-gap` is the name of the old property, which has been deprecated